electron-app/package.json

```diff
@@ -33,48 +33,48 @@
     }
   },
   "dependencies": {
-    "@theia/callhierarchy": "^1.9.0",
-    "@theia/console": "^1.9.0",
-    "@theia/core": "^1.9.0",
-    "@theia/debug": "^1.9.0",
-    "@theia/editor": "^1.9.0",
-    "@theia/editor-preview": "^1.9.0",
-    "@theia/electron": "^1.9.0",
-    "@theia/file-search": "^1.9.0",
-    "@theia/filesystem": "^1.9.0",
-    "@theia/getting-started": "^1.9.0",
-    "@theia/git": "^1.9.0",
-    "@theia/keymaps": "^1.9.0",
-    "@theia/markers": "^1.9.0",
-    "@theia/messages": "^1.9.0",
-    "@theia/metrics": "^1.9.0",
-    "@theia/mini-browser": "^1.9.0",
-    "@theia/monaco": "^1.9.0",
-    "@theia/navigator": "^1.9.0",
-    "@theia/outline-view": "^1.9.0",
-    "@theia/output": "^1.9.0",
-    "@theia/plugin-dev": "^1.9.0",
-    "@theia/plugin-ext": "^1.9.0",
-    "@theia/plugin-ext-vscode": "^1.9.0",
-    "@theia/preferences": "^1.9.0",
-    "@theia/preview": "^1.9.0",
-    "@theia/process": "^1.9.0",
-    "@theia/scm": "^1.9.0",
-    "@theia/scm-extra": "^1.9.0",
-    "@theia/search-in-workspace": "^1.9.0",
-    "@theia/task": "^1.9.0",
-    "@theia/terminal": "^1.9.0",
-    "@theia/timeline": "^1.9.0",
-    "@theia/typehierarchy": "^1.9.0",
-    "@theia/userstorage": "^1.9.0",
-    "@theia/variable-resolver": "^1.9.0",
-    "@theia/vsx-registry": "^1.9.0",
-    "@theia/workspace": "^1.9.0",
+    "@theia/callhierarchy": "^1.10.0",
```

Comment:
With recent changes in the framework, where we now publish using exact versions for our own dependencies, I think it's better to also target a specific version here. i.e. remove the "^" for the `@theia` dependencies. 

With do this already in the main example application: 

https://github.com/eclipse-theia/theia/blob/master/examples/browser/package.json#L17-L54

Reply:
unless... We keep it as-is and then I think it might cause issues when the version (e.g. `1.10.0`) is no longer the most recent `latest`, potentially reminding us to update this app to use the latest? ;)